src/module/os.c

```diff
@@ -109,6 +109,20 @@ void processExit(WrenVM* vm) {
   uv_stop(getLoop());
 }
 
+// chdir_(dir)
+void processChdir(WrenVM* vm)
+{
+  wrenEnsureSlots(vm, 1);
```

Comment:
I'm not even sure what this does.

Reply:
You have to make room in the VM slots for a return value.  Supposedly this is always needed even if the incoming number of slots (params, etc) has you covered.  IE, an instance method should always have 1 slot since slot 0 is the pointer to the object itself, but I've been told this is still always necessary.

Reply:
In this case 1 is correct since we're not responsible for incoming slots - the VM has already reserved space for them.. we're only responsible for outgoing/scratchpad... - though I've been told even if the incoming slot space should be sufficient it's still a good idea to declare this explicitly.

Reply:
`chdir` does not return any value, so I'll remove it.

Reply:
It's needed in case of an error, though I suppose you could theoretically move it inside the conditional though but traditionally it's the first thing in a function IIRC.